app/code/Magento/Review/Helper/Data.php

```diff
@@ -51,7 +53,7 @@ public function __construct(
      * @param string $origDetail
      * @return string
      */
-    public function getDetail($origDetail)
+    public function getDetail(string $origDetail): string
```

Comment:
Please remove param strict types to ensure backward compatibility (i.e. $origDetail can be currently passed as a \Magento\Framework\Phrase by third-party extension)